DuggaSys/gitcommitService.php

```diff
@@ -263,6 +262,7 @@ function clearGitFiles($cid) {
 	// getCommit: Gets the latest commit from a URL using DOM
 	//--------------------------------------------------------------------------------------------------
 
+	//this is useless, do bfs($url,$cid,"GETCOMMIT") instead where applicable, leaving for now due to it being referenced elesewhere
```

Comment:
This function is useless, yet its changed?

Reply:
This function was deemed useless as it would always return an empty string. Considering the function tries to get the latest commit by essentially loading the github website and fetching it from the element that says it, it could reasonably be assumed that this code was simply outdated. Any changes to the function are simply leftovers from testing.

It has been replaced by the new bfs() functionality with GETCOMMIT, but it is still referenced elsewhere so it was deemed reasonable to leave it there for now.